lib/cog_api/fake/relay_groups.ex

```diff
@@ -95,13 +95,18 @@ defmodule CogApi.Fake.RelayGroups do
     {:ok, Server.update(RelayGroup, relay_group.id, relay_group)}
   end
 
-  def add_bundle(_, _, %Endpoint{token: nil}), do: Endpoint.invalid_endpoint
-  def add_bundle(%{name: name}, %{bundle: bundle_name}, %Endpoint{token: _}) do
-    bundle = Server.show_by_key(Bundle, :name, bundle_name)
+  def add_bundles(_, _, %Endpoint{token: nil}), do: Endpoint.invalid_endpoint
+  def add_bundles(%{name: name}, %{bundles: bundle_names}, %Endpoint{token: _}=endpoint) do
     relay_group = Server.show_by_key(RelayGroup, :name, name)
-    add_bundle(bundle, relay_group)
+    bundle_ids = Enum.map(bundle_names, &Server.show_by_key(Bundle, :name, &1))
+    |> Enum.map(&Map.fetch!(&1, :id))
+    add_bundles(relay_group.id, bundle_ids, endpoint)
+  end
+  def add_bundles(id, bundle_ids, %Endpoint{token: _}=endpoint) when is_list(bundle_ids) do
```

Comment:
Yes. This function recurses. When a single id is passed the next function head is invoked.